api/src/pcapi/routes/serialization/venues_serialize.py

```diff
@@ -324,6 +324,16 @@ class EditVenueBodyModel(BaseModel, AccessibilityComplianceMixin):
     reimbursementPointId: int | None
     shouldSendMail: bool | None
 
+    @validator("venueTypeCode")
+    @classmethod
+    def validate_venue_type_code(cls, venue_type_code: str | None) -> str | None:
+        if venue_type_code is not None:
+            try:
+                offerers_models.VenueTypeCode[venue_type_code]
+            except KeyError:
+                raise ValueError("Invalid venue type code")
+        return venue_type_code
+
```

Comment:
Dans ce cas, on supprimer ce validateur, non ? J'ai testé en local et les tests passent.

Reply:
Oui évidemment, j'ai été un peu trop vite

Reply:
Mais en fait, avec le type d'enum à la place de str, j'ai :
`Please run 'yarn generate:api:client:local' and add the changes to the commit(s) that change the API.`
dans la CI, donc potentiellement un impact sur le front que je ne sais mesurer

Reply:
La génération du code front n'est plus fait automatiquement par la CI maintenant, c'est au développeurs de lancer la commande indiquée. Le changement a dû se faire pendant tes vacances.